extensions/wikia/ArticleVideo/styles/_video-controls.scss

```diff
@@ -0,0 +1,159 @@
+
+.oo-player-container {
+	.oo-scrubber-bar-container {
+		height: 5px; // Adjustment for smaller height on controlbar
+
+		.oo-scrubber-bar-padding {
+			.oo-scrubber-bar {
+				height: $video-scrubber-height;
+			}
+
+			.oo-playhead-padding {
+				.oo-playhead {
+					background: $video-player-head-color;
+					border-radius: $video-player-head-size;
+					border: 2px solid #fff;
+					height: $video-player-head-size;
+					width: $video-player-head-size;
+				}
+
+				.oo-playhead-scrubbing {
+					background: $video-player-head-color;
+					border-radius: $video-player-head-size-scrubbing;
+					border: 3px solid #fff;
+					height: $video-player-head-size-scrubbing;
+					width: $video-player-head-size-scrubbing;
+				}
+			}
+		}
+	}
+
+	.oo-control-bar {
+		.oo-logo, .oo-closed-caption, .oo-more-options {
+			display: none !important;
```

Comment:
This wouldn't be necessary if we e.g. disabled the closed captions using `closedCaptionOptions` in `skin.json`